src/index.html

```diff
@@ -5,8 +5,423 @@
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price-description">
+          <p class="card__price">Price:</p>
+
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <a href="#"
+          class="card__buy-link"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article
+        class="card catalog__card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <h2 class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__id">
+          Product code: 195434
+        </p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price-description">
+          <p class="card__price">Price:</p>
+
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <a href="#"
+          class="card__buy-link"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </article>
+
+      <article
+        class="card catalog__card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <h2 class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__id">
+          Product code: 195434
+        </p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price-description">
+          <p class="card__price">Price:</p>
+
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <a href="#"
+          class="card__buy-link"
+          data-qa="card-hover"
```

Comment:
add data-qa="card-hover" (not just hover) to the link Buy **only** inside the first card